doc/Glossary/index.md

```diff
@@ -5,74 +5,236 @@ permalink: /:path/
-| Transport channel                                       | A Kaa client component responsible for communication with a Kaa server. Refer to Transports for more information.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
-| U                                                       |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
-| User verifier                                           | A server component that handles the user verification. Refer to User verifiers for more information.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          
```

Comment:
"Actor gateway" is in fact a component, not a type of deployment. And you should explain what it is by telling what it does, not when it's used. You can mention that is is especially useful under these and that circumstances in 2-nd or 3-rd phrase.